src/components/SimpleTable.tsx

```diff
@@ -125,49 +30,108 @@ export interface IProps {
-    {hasError && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-error" />
-        </Cell>
-      </Row>
-    )}
-    {!hasError && content.length === 0 && (
-      <Row>
-        <Cell>
-          <Translation id="page-stablecoins-table-loading" />
-        </Cell>
-      </Row>
-    )}
-    {content.map((row, idx) => {
-      const { name, marketCap, image, type, url } = row
-      const rowContent = (
-        <>
-          <Cell>
-            {image && <StyledImage src={image} alt="" />}
-            {name}
-          </Cell>
-          <Cell>{marketCap}</Cell>
-          <Cell>{type}</Cell>
-        </>
-      )
+const SimpleTable: React.FC<IProps> = ({ columns, content, hasError }) => {
+  const [textColor] = useToken("colors", ["text"])
+
+  return (
+    <Table
+      variant="unstyled"
+      my={8}
+      borderRadius="2px"
+      border={`1px solid ${textColor}`}
+      bg="background"
+      mb={8}
+      minW="720px"
+    >
+      <Thead bg="ednBackground" color="text200">
+        <Tr boxShadow={`0 1px 1px ${textColor}`} mb="1px">
+          {columns.map((column, idx) => (
+            <Th
+              key={idx}
+              py={5}
+              fontSize="md"
+              fontWeight="normal"
+              letterSpacing="normal"
+            >
+              {column}
+            </Th>
+          ))}
 
-      return url ? (
-        <LinkRow key={idx} to={url}>
-          {rowContent}
-        </LinkRow>
-      ) : (
-        <Row key={idx}>{rowContent}</Row>
-      )
-    })}
-  </Table>
-)
+          {content && content[0]?.url && (
+            <Th p={5} fontSize="md" fontWeight="normal" textAlign="end">
+              ↗
+            </Th>
+          )}
+        </Tr>
+      </Thead>
+      <Tbody>
+        {hasError && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-error" />
+            </Td>
+          </Tr>
+        )}
+        {!hasError && content.length === 0 && (
+          <Tr p={4}>
+            <Td colSpan={4}>
+              <Translation id="page-stablecoins-table-loading" />
+            </Td>
+          </Tr>
+        )}
 
+        {content.map(({ name, marketCap, image, type, url }, idx) => (
+          <Tr
+            as={url ? LinkBox : undefined}
+            display="table-row"
+            href={url}
+            key={idx}
+            color="white"
+            textDecoration="none"
+            boxShadow={`0 1px 1px ${textColor}`}
+            _hover={{
+              textDecoration: "none",
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+              cursor: url ? "pointer" : "default",
+            }}
+            _focus={{
+              boxShadow: `0 0 1px ${textColor}`,
+              bg: "primary200",
+              color: "black300",
+            }}
+          >
+            <Td display="flex" alignItems="center">
```

Comment:
I think we need to apply these styles to all the `Td`s since I don't see them aligned on the center.

![image](https://user-images.githubusercontent.com/468158/209213268-c019574a-b4e4-49d1-9263-f9b1c294eee9.png)

Reply:
Done!